app/views/transactions/index.html.erb

```diff
@@ -1,8 +1,9 @@
 <p style="color: green"><%= notice %></p>
-
-<h4>Transactions</h4>
-<div class="d-flex justify-content-between px-4 fs-3" >
+<div style="width: 100%">
 <%= link_to "Back", categories_path, class:"btn btn-sucess mb-3 bg-primary text-white fs-4"%>
+</div>
+<div class="d-flex justify-content-between px-4 fs-3" >
+<h4>Transactions</h4>
 <%= "Total amount:  #{@total_amount}" %>
 </div>
 <div id="transactions">
```

Comment:
- [ ] Good job trying to fix some of changes mentioned by previous reviewer. However, I noticed that there is something you should improve again about the styling as the previous reviewer mentioned and to match the template design. You are are supposed to use a `ShowLeftArrow` icon instead of `Back` text. For more details about using icons in rails, just **[refer this link](https://www.youtube.com/watch?v=nfLGQd0zHo4)**

| Template design | Your Design |
| ----------------------- | ------------------ |
| ![Details](https://user-images.githubusercontent.com/68796232/242135274-7f9b1687-68bc-44ea-ae57-f80c4d5a9377.png) | ![Screenshot from 2023-05-31 05-30-15](https://github.com/V0pe/Payday/assets/83042742/7299cde9-9139-46f3-a7e1-45320549996e) |

Reply:
@tresorsawasawa Thanks, I have fix it
![image](https://github.com/V0pe/Payday/assets/61356487/39e7fbff-e04d-4c96-999f-c6ba87acae29)